hops/hdfs.py

```diff
@@ -66,6 +66,13 @@ def check(self, args):
         return self.walk(self.root, os.listdir(self.root), self.hdfs_root, hdfs.ls(self.hdfs_root))
 
 
+def project_id():
+    """
```

Comment:
Move "HOPSWORKS_PROJECT_ID" to a string constant in case we change it in the future or need it in more places.

Reply:
Thanks, do we need to remote the `:` from the Args here then? https://github.com/logicalclocks/hops-util-py/blob/branch-0.9/hops/hdfs.py#L162

Reply:
I think as long as you follow the patterns of: 
`
"Args:
     ....

Returns:
....
Attributes:

   ...
`
it's fine, seems that some editors add two colons for the args and some use one, didn't think about it before.

Reply:
> Move "HOPSWORKS_PROJECT_ID" to a string constant in case we change it in the future or need it in more places.

We do the same with other env vars as well, [example](https://github.com/logicalclocks/hops-util-py/blob/branch-0.9/hops/hdfs.py#L27). Will fix them.

Reply:
> I think as long as you follow the patterns of:
> `
> "Args:
> ....
> 
> Returns:
> ....
> Attributes:
> 
> ...
> `
> it's fine, seems that some editors add two colons for the args and some use one, didn't think about it before.

It's just that intellij does not add the `:` so wanted to make sure